js/script.js

```diff
@@ -16,8 +20,153 @@ function toggleOpenMenu() {
   }
 }
 
-hamburger.addEventListener('click', toggleOpenMenu);
+function createLanguagesUlElement(languages) {
+  let markup = '<ul>';
+  languages.forEach((language) => {
+    markup += `<li>${language}</li>`;
+  });
+  markup += '</ul>';
+  return markup;
+}
+
+const markupModal = (projectId) => {
+  const project = projects[projectId];
+  return `<div class="project-margin">
+            <div class="project-modal-container">
+              <div class="modal-header">
+                <h2>${project.name}</h2>
+                <img src="./assets/img/close-icon.png" class="close" alt="Close Modal">
+              </div>
+              ${createLanguagesUlElement(project.languages)}
```

Comment:
- [ ] [REQUIRED] You have indeed done a great job @emmiiorji , your popup follows the design for the mobile version from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471), however, the popup for the desktop language buttons needs a resize. The current language button on your design looks too big which does make the UI look professional. Kindly reduce the size and ensure your design looks more like the figma template popup.
### Your Design
![Screenshot from 2022-04-14 19-44-22](https://user-images.githubusercontent.com/58692590/163458383-d2c8f90c-9cca-4392-b69f-7647b165010c.png)
### Figma Template 2
![Screenshot from 2022-04-14 19-44-39](https://user-images.githubusercontent.com/58692590/163458434-e216af31-6ca7-467f-89bb-f8344d2fd7d1.png)